db/migrate/20190911124324_create_login_activities.rb

```diff
@@ -0,0 +1,23 @@
+class CreateLoginActivities < ActiveRecord::Migration
+  def change
+    create_table :login_activities do |t|
+      t.string :scope
+      t.string :strategy
+      t.string :identity
```

Comment:
I'd put `null: false` wherever I could, but I don't know if that breaks things.

Reply:
Yeah, this is the default migration from the gem, so I'm going to leave it in this instance.